lib/ffmpeg/qsv/encoder.py

```diff
@@ -63,6 +63,7 @@ def encparams(self):
       f"{super().encparams}{self.ldb}"
       f"{self.iqfactor}{self.bqfactor}"
       f"{self.iqoffset}{self.bqoffset}"
+      f"{self.seek}"
```

Comment:
Since this property is in the base class and is generic, please add this part to the base class `encparams`, instead.